### PR TITLE
[xthor] Future fix: New domain name for xthor

### PR DESCRIPTION
**Pull request by Tawagot0**
This is a future fix for the xthor domain name.
The current domain name expires in one month and will no longer be accessible.
I will mention @garfield69 once the change is made.

### CODE REVIEW

File: src/Jackett.Common/Indexers/Xthor.cs [hunk 1]
```diff
@@ -24,7 +24,7 @@ namespace Jackett.Common.Indexers
     /// </summary>
     public class Xthor : BaseCachingWebIndexer
     {
-        private static string ApiEndpoint => "https://api.xthor.to/";
+        private static string ApiEndpoint => "https://api.xthor.tk/";
 
         public override string[] LegacySiteLinks { get; protected set; } = new string[] {
             "https://xthor.bz/",
```
Comment (garfield69) [inline src/Jackett.Common/Indexers/Xthor.cs]: please add https://xthor.to/ to the LegacySiteLinks block.

File: src/Jackett.Common/Indexers/Xthor.cs [hunk 2]
```diff
@@ -27,7 +27,7 @@ public class Xthor : BaseCachingWebIndexer
         private static string ApiEndpoint => "https://api.xthor.tk/";
 
         public override string[] LegacySiteLinks { get; protected set; } = new string[] {
-            "https://xthor.bz/",
+            "https://xthor.to/",
         };
```
Comment (garfield69) [inline src/Jackett.Common/Indexers/Xthor.cs]: Ah no, leave the old one there.
the purpose of this block is to automatically replace any config's with one of these old-sitelinks with the new sitelink defined in this indexer.
this way users with existing configs do not need to manually edit their configs to update ;-)

**Comment by Tawagot0** [comment]
Done, thanks

File: src/Jackett.Common/Indexers/Xthor.cs (hunk 2 above)
Comment (Tawagot0) [reply src/Jackett.Common/Indexers/Xthor.cs]: Oh ok, i understood thanks!
This is fixed, the old domain name is no longer accessible, can you merge?

**Comment by virtualfrancky** [comment]
Hi I am trying to find this file on my Quickbox installation, but it does not exist, Quickbox support says they use the standalone version, does this make any difference ?

**Comment by garfield69** [comment]
@virtualfrancky  just wait for the release to be published. this indexer is written in c# so is part of the Jackett binaries.

@Tawagot0 Thank you.

**Pull request merged**